core/test/oracle/Voting.js

```diff
@@ -115,7 +115,11 @@ contract("Voting", function(accounts) {
     // Voters can alter their commits.
     const newPrice = getRandomSignedInt();
     const newSalt = getRandomUnsignedInt();
-    const newHash = web3.utils.soliditySha3(newPrice, newSalt);
+    const newHash = web3.utils.soliditySha3(
```

Comment:
@mrice32 Added an example of overriding soliditySha3 type autodetection, but will hold off on making sweeping changes across repo until another PR